### PR TITLE
View tournaments

### LINKED ISSUE #42: View Tournaments
What has to be done:
Create a component that will be used for viewing all tournaments.

Why it has to be done:
So that users can search all tournaments and find the tournament that they want to view.

When the task is done:
The user can view a list of tournaments which they can filter on tournament settings and can search for certain tournaments.
The tournament view should have pagination.

### PR DESCRIPTION
**Pull request by MindHawk**
Fixed #42

### CODE REVIEW

File: src/views/tournament/AllTournamentsView.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            selectedTournament: {}
+        };
+    }
+
+    getAllTournaments() {
+        const body = "query {tournaments{name}}";
+        const vars = "{}";
+        ApiCommunication.graphQlCallPost(body, vars)
+            .then(response => this.state.tournaments = response.data.data.tournaments)
+            .catch(error => console.log(error));
```
Comment (lmtjanssen) [inline src/views/tournament/AllTournamentsView.jsx]: logging error is not master code

Comment (MindHawk) [reply src/views/tournament/AllTournamentsView.jsx]: image

Comment (MindHawk) [reply src/views/tournament/AllTournamentsView.jsx]: image

Comment (MindHawk) [reply src/views/tournament/AllTournamentsView.jsx]: 🤔

Comment (MindHawk) [reply src/views/tournament/AllTournamentsView.jsx]: OK made it console.error

Comment (chf3kl9) [reply src/views/tournament/AllTournamentsView.jsx]: Response on: "Console.log is not allowed, but is in the master, so it is allowed".
By this reasoning:
School shootings zijn verboden, maar gebeuren, dus ze zijn legaal.
I guess we both understand that this reasoning is wrong

File: src/views/tournament/AllTournamentsView.jsx [hunk 2]
```diff
@@ -17,8 +17,8 @@ class AllTournamentsView extends Component {
         const body = "query {tournaments{name}}";
         const vars = "{}";
         ApiCommunication.graphQlCallPost(body, vars)
-            .then(response => this.state.tournaments = response.data.data.tournaments)
-            .catch(error => console.log(error));
+            .then(response => this.setState({tournaments: response.data.data.tournaments}))
+            .catch(error => console.error(error));
```
Comment (lmtjanssen) [inline src/views/tournament/AllTournamentsView.jsx]: ...

**Pull request merged**